docs/reference/rollup/apis/get-job.asciidoc

```diff
@@ -101,7 +101,23 @@ Which will yield the following response:
             "pages_processed" : 0,
             "documents_processed" : 0,
             "rollups_indexed" : 0,
-            "trigger_count" : 0
+            "trigger_count" : 0,
+            "bulk_failures": 0,
+            "bulk_latency_in_ms": {
```

Comment:
Can we simplify this to:
```
"bulk_time_in_ms": 0,
"bulk_total": 0,
"search_time_in_ms": 0,
"search_total": 0
````
?
I don't think we need more than the total time and the number of invocations.

Reply:
Sure, I can simplify these.  :)